src/layer_normalization.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import {add, sub, mul, div, pow} from './binary.js';
+import {reduceMean} from './reduce.js';
+import {reshape} from './reshape.js';
+import {sqrt} from './unary.js';
+import {Tensor, Scalar} from './lib/tensor.js';
+import {transpose} from './transpose.js';
+import {validateLayerNormalizationParams} from './lib/validate-input.js';
+
+/**
+ * Normalize the tensor values of input features using
+ * [layer-Normalization](https://arxiv.org/abs/1607.06450)
+ * @param {Array} axes
+ * @param {Tensor} input
+ * @param {MLBatchNormalizationOptions} [options]
+ * @return {Tensor}
+ */
+export function sortByValue(axes) {
```

Comment:
The function description isn't align with the below **sortByValue** function. Please add each description for each function.

And I suggest renaming `sortByValue` by `getIndexOfSortedValue`, it would be understandable.